rocky/reports/views/aggregate_report.py

```diff
@@ -107,6 +110,21 @@ class ExportSetupAggregateReportView(
     current_step = 4
     report_type = AggregateOrganisationReport
 
+    def post(self, request, *args, **kwargs):
+        selected_plugins = request.POST.getlist("plugin", [])
+
+        client = get_katalogus(self.organization.code)
+        for selected_plugin in selected_plugins:
+            try:
+                client.enable_boefje_by_id(selected_plugin)
+            except HTTPError:
```

Comment:
I hate that we bubble up the http errors from the client to here. Lets move that away asap, the katalogusclient already has some localized exceptions for other issues.